group_vars/all.yml

```diff
@@ -173,3 +173,8 @@ shift_stack: false
 #      ram: 16384
 #      disk: 40
 #      vcpus: 4
+
```

Comment:
Asma and Masco

May be defining 'osp_rhel_mapping' like below in group_vars, with a comment saying user can change the mapping for any specific use case and if the lab supports the corresponding rhel version
    osp_rhel_mapping:
      10: 7.7
      13: 7.7
      14: 7.7
      15: 8.0
      16: 8.1
      16.1: 8.2

Later you can use this mapping in setup_undercloud.yml like below
http://paste.openstack.org/show/795071/

Reply:
Anil,
Maybe we can update about the osp_rhel_mapping in the README so that the user can update here [1] depending on the use-case (optional) otherwise the tasks are automated depending on osp_release. This way it avoids adding osp_rhel_mapping in the group_vars for each osp_release. Please share your thoughts?

[1] https://github.com/redhat-performance/jetpack/pull/205/files#diff-ee7b44b9b73d4c691251cbf5c8dff5c5R179